src/components/Featurecard Small/FeatureCardSmall.js

```diff
@@ -0,0 +1,41 @@
+import StyledSection from '../styled-components/StyledSection';
+import StyledDiv from '../styled-components/StyledDiv';
+import StyledUL from '../styled-components/StyledUL';
+import StyledList from '../styled-components/StyledList';
+import StyledInput from '../styled-components/StyledInput';
+import Typography from '../styled-components/Typography';
+import todos from '../../services/static-todos.json';
+
+export default function FeatureCardSmall() {
+	return (
+		<StyledSection variant="small-grey">
+			<StyledDiv variant="cardtitle">
+				<StyledInput
+					variant="featurecheckbox"
+					id="featurecheckbox"
+					name="featurecheckbox"
+					type="checkbox"
+				/>
+				<Typography variant="h2">Featurename</Typography>
+				<StyledDiv variant="statusbox">
+					<Typography variant="p">Status</Typography>
+				</StyledDiv>
+			</StyledDiv>
+			<StyledUL variant="list-in-card">
+				{todos.slice(0, 3).map(todo => {
+					return (
+						<StyledList key={todo.id} variant="featurelist-small">
+							<StyledInput
+								variant="todocheckbox"
+								id="todocheckbox"
+								name="todocheckbox"
+								type="checkbox"
+							/>
+							{todo.name}
+						</StyledList>
+					);
+				})}
```

Comment:
Why are you using the slice() method here? 

Check out [what slice does](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice), it doesn't seem like this method is what you're looking for